src/bindings/python/src/openvino/runtime/opset1/ops.py

```diff
@@ -362,12 +362,15 @@ def convert(
     :param name: Optional name for the output node.
     :return: New node performing the conversion operation.
     """
-    if not isinstance(destination_type, str):
-        destination_type = get_element_type_str(destination_type)
+    _destination_type = None  # type: Union[str, Type]
+    if isinstance(destination_type, get_args(NumericType)):
+        _destination_type = get_element_type_str(destination_type).lower()
+    else:
+        _destination_type = destination_type
     return _get_node_factory_opset1().create(
         "Convert",
         [as_node(data)],
-        {"destination_type": destination_type.lower()},
+        {"destination_type": _destination_type},
```

Comment:
Just to ensure, both `str` and `ov.Type` was already supported as attribute value to the node factory?